llama_index/llms/azure_openai.py

```diff
@@ -88,6 +91,10 @@ def __init__(
             "azure_endpoint", azure_endpoint, "AZURE_OPENAI_ENDPOINT", ""
         )
 
+        # Use the custom httpx client if provided.
+        # Otherwise a default one will be created.
+        self._http_client = http_client or httpx.Client()
```

Comment:
would it be better to leave it as `None` and let the `openai` package handle creating the client?

Reply:
Ok, I understand👍

Reply:
This seems to be causing a problem in 0.9.1? #8944

Reply:
Hi, thank for notify me. I think that the fixing is in #8955